client/ayon_core/hosts/max/api/pipeline.py

```diff
@@ -52,11 +52,8 @@ def install(self):
 
         self._has_been_setup = True
 
-        def context_setting():
-            return lib.set_context_setting()
-
         rt.callbacks.addScript(rt.Name('systemPostNew'),
-                               context_setting)
+                               on_post_open)
```

Comment:
This being used on `new` to me is confusing that it calls a function named `on_post_open`. Why isn't the function named `on_new` or `on_new_scene` or alike?

Reply:
updated!